docs/en/writing-running-appium/android/android-appbundle.md

```diff
@@ -0,0 +1,51 @@
+## How To Test Android App Bundle
+
+Google has released [Android App Bundle](https://developer.android.com/platform/technology/app-bundle/) feature.
+`.aab` file is generated by the feature. We must upload the file to Google Play Store.
+
+We can get distributed apk files from the `.aab` file via [bundletool](https://github.com/google/bundletool). Using the generated files, we can test against release module. After [1](https://github.com/appium/appium-adb/pull/367) and [2](https://github.com/appium/appium-base-driver/pull/271), you can run Appium tests against `.apks` file. The commits are available over Appium 1.9.2.
+
+## How to run tests
+
+1. Export `bundletool.jar` in your path
+    - Appium search the `bundletool.jar` in your local. Make sure you can find the path with `which 'bundletool.jar'`. If you can't find it, please set the path correctly.
+2. Generate `.apks` from `.aab`
```

Comment:
I believe this might be also possible with Android Studio UI

Reply:
since version 3.2

Reply:
Generate the `.apks` file from the `.aab` file